### PR TITLE
Undertone 24910 video in prebid

### PR DESCRIPTION
**Pull request by AnnaPerion**
##Type of change
- [x] Feature
- [x] Does this change affect user-facing APIs or examples documented on http://prebid.org?

## Description of change
- Adding support for video ads in Undertone adapter

### CODE REVIEW

**Comment by bretg** [comment]
@AnnaPerion - please fix your indentation -- it failed the lint stage.

**Comment by patmmccann** [comment]
@AnnaPerion this appears to break undertone bidding on multiformat units. Please see first in content unit at https://cafedelites.com/best-chimichurri-chicken

Your volume on this unit is down nearly 100% with us since this commit was merged.

File: modules/undertoneBidAdapter.js [hunk 1]
```diff
@@ -120,8 +122,20 @@ export const spec = {
         sizes: bidReq.sizes,
         params: bidReq.params
       };
+      const videoMediaType = deepAccess(bidReq, 'mediaTypes.video');
```
Comment (patmmccann) [inline modules/undertoneBidAdapter.js:125]: this should potentially say &  deepAccess(bidReq, 'mediaTypes.video.context') != 'outstream or something of the sort

Comment (oranperion) [reply modules/undertoneBidAdapter.js:125]: @patmmccann i believe our team already reached out to you via email, however please reach out to pubams@undertone.com for further discussion/ next steps

Comment (patmmccann) [reply modules/undertoneBidAdapter.js:125]: @oranperion this is a community issue, not just our account. What is the status of this? Should we revert this pr as it introduces the bug or are you planning a patch?

Comment (oranperion) [reply modules/undertoneBidAdapter.js:125]: @patmmccann issue already been solved on backend, there is already high fill rate on the placement id 2002001, and an email about this information been sent to you and your company last week, where a member of your company - Isaiah Ward <iward@cafemedia.com> conform issue resolved

**Comment by patmmccann** [comment]
Thanks for looping in the community here

File: modules/undertoneBidAdapter.js (hunk 1 above)
Comment (patmmccann) [reply modules/undertoneBidAdapter.js:125]: Thanks for looping in the community here, glad all is resolved!

**Pull request merged**